### PR TITLE
Fix: show loading indicator after submitting an external connection URL

### LINKED ISSUE #595: Auth wizard: No indicator that anything is happening after submitting an external connection URL
**Is your enhancement related to a problem? Please describe.**
There is no indicator that anything is happening after you hit "Authorize Connection" - currently have to click/activate the button, enter key doesn't work correctly but when it does it exacerbates the issue, see #594.

**Describe the solution you'd like**
Since there's already a spot where error messages show, seems to me a spinner should show there with a message saying what's happening, e.g. "Checking for Distributor" or something like that.

**Designs**
Message area is where it says "Unable to connect to site"

Screen Shot 2020-06-11 at 8 46 07 PM

**Describe alternatives you've considered**
Spinner inside input field

**Additional context**
Noted by @dinhtungdu

### PR DESCRIPTION
**Pull request by dinhtungdu**
### Description of the Change
Fixes #595.

Show a spinner and a message indicates that we're checking the connection. 

### Benefits
Make the UI more interactive.

### Verification Process
Create a new connection using Auth Wizard, see the spinner and message after submitting the external site URL.

### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://github.com/10up/distributor/blob/develop/CONTRIBUTING.md) document.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: includes/classes/Authentications/WordPressBasicAuth.php [hunk 1]
```diff
@@ -79,6 +79,10 @@ public static function credentials_form( $args = array() ) {
 				<a href="#" class="manual-setup-button">
 					<?php esc_html_e( 'Manually Set Up Connection', 'distributor' ); ?>
 				</a>
+				<div class="dt-wizard-status">
+					<span class="spinner is-active"></span>
+					<span><?php esc_html_e( 'Checking the connection..', 'distributor' ); ?></span>
```
Comment (helen) [inline includes/classes/Authentications/WordPressBasicAuth.php]:
```suggestion
					<span><?php esc_html_e( 'Checking the connection...', 'distributor' ); ?></span>
```

Should be a full 3-dot ellipsis :)

**Review by helen — CHANGES_REQUESTED** [review]
Have one string tweak and one other thing I noticed - the checking message shows even with an empty URL and then gets stuck there. I think it probably shouldn't show until it's past the URL checking a few lines later.

**Comment by dinhtungdu** [comment]
@helen Thanks for the review! Now the status only shows after URL and title are validated.

**Review by helen — APPROVED** [review]
Works great now!

**Pull request merged**